### PR TITLE
Items_Receiving_quantity Update

### PR DESCRIPTION
**Pull request by odiea**
Items Receiving quantity update

### CODE REVIEW

File: database/receiving.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE ospos_items SET receiving_quantity = 1 WHERE receiving_quantity = 0;
```
Comment (daN4cat) [inline database/receiving.sql]: all good but this change should be part of the usual database update script

Comment (odiea) [inline database/receiving.sql]: I am not that advanced using github to do that plus I thought it was such a small update.

Comment (daN4cat) [inline database/receiving.sql]: I'll sort it out layer

Comment (odiea) [inline database/receiving.sql]: Thanks. I don't know the magic behind updating the database.

**Comment by daN4cat** [comment]
Merged

**Comment by odiea** [comment]
I missed part of the code needed to change the Receivings Report to show properly. How would I submit it without adding all the changes you already committed. I don't know how to rebase a branch on github.

**Comment by dvidic** [comment]
Am I missing something. When I receive items via Receivings and enter receiving quantity, item have 0 on stock. On Item info theres a number under receiving quantity, but under stock I get 0.

**Pull request closed without merging**